utility_network/display-utility-associations/src/main/java/com/esri/samples/display_utility_associations/DisplayUtilityAssociationsSample.java

```diff
@@ -0,0 +1,244 @@
+      associationsOverlay.setRenderer(new UniqueValueRenderer(Arrays.asList("AssociationType"), Arrays.asList(attachmentUniqueValue, connectivityUniqueValue), "", null));
+  
+      // create a map with the topographic vector basemap
+      ArcGISMap map = new ArcGISMap(Basemap.createTopographicVector());
+      
+      // create a viewpoint to focus on the utility networks'extent
+      Viewpoint viewPoint = new Viewpoint(41.8057655, -88.1489692, 23);
+  
+      // set initial arcgis map extent
+      map.setInitialViewpoint(viewPoint);
+      // create a map view and set its map
+      mapView = new MapView();
+      mapView.setMap(map);
+      mapView.getGraphicsOverlays().add(associationsOverlay);
+      
+      // create the utility network
+      utilityNetwork = new UtilityNetwork("https://sampleserver7.arcgisonline.com/arcgis/rest/services/UtilityNetwork/NapervilleElectric/FeatureServer");
+  
+      // load utility network async and get all of the edges and junctions in the network
+      utilityNetwork.loadAsync();
+      utilityNetwork.addDoneLoadingListener(() -> {
+        List<UtilityNetworkSource> networkSources = utilityNetwork.getDefinition().getNetworkSources();
+    
+        for (UtilityNetworkSource nSource : networkSources)
+          // check an add all edges that are not subnet lines to the map.
+          if (nSource.getSourceType() == UtilityNetworkSource.Type.EDGE && nSource.getSourceUsageType() != UtilityNetworkSource.UsageType.SUBNET_LINE) {
+            mapView.getMap().getOperationalLayers().add(new FeatureLayer(nSource.getFeatureTable()));
+          }
+          // add all junctions to the map
+          else if (nSource.getSourceType() == UtilityNetworkSource.Type.JUNCTION) {
+            mapView.getMap().getOperationalLayers().add(new FeatureLayer(nSource.getFeatureTable()));
+          }
+    
+        // add association graphics at the initial view point
+        addAssociationsGraphics();
+        // listen for navigation changes
+        mapView.addNavigationChangedListener((event) -> {
+          addAssociationsGraphics();
+        });
+      });
+      
+      // create a controls box
+      VBox controlsVBox = new VBox();
+      controlsVBox.setBackground(new Background(new BackgroundFill(Paint.valueOf("rgba(255,255,255,0.5)"), CornerRadii.EMPTY,
+        Insets.EMPTY)));
+      controlsVBox.setPadding(new Insets(10.0));
+      controlsVBox.setSpacing(5);
+      controlsVBox.setMaxSize(100, 50);
+      controlsVBox.setAlignment(Pos.BASELINE_LEFT);
+      controlsVBox.getStyleClass().add("panel-region");
+      controlsVBox.getChildren().addAll(attachmentLabel, attachmentImageView, connectivityLabel, connectivityImageView);
+      
+      // add the scene view and controls to the stack pane
+      stackPane.getChildren().addAll(mapView, controlsVBox);
+      StackPane.setAlignment(controlsVBox, Pos.TOP_LEFT);
+      StackPane.setMargin(controlsVBox, new Insets(10, 0, 0, 10));
```

Comment:
I've had a go, it could look something like the suggestion below.
All you need to do is to add objects and specify the row/column.
We need to add a line at the top of the file to link the CSS file, that's where the rest of the styling will happen.

```suggestion
      // create a legend
      GridPane gridPane = new GridPane();
      gridPane.getStyleClass().add("grid-pane");
      gridPane.getColumnConstraints().addAll(Arrays.asList(new ColumnConstraints(25), new ColumnConstraints(100)));
      gridPane.add(attachmentImageView, 0, 0);
      gridPane.add(attachmentLabel, 1, 0);
      gridPane.add(connectivityImageView, 0, 1);
      gridPane.add(connectivityLabel, 1, 1);

      // add the scene view and legend to the stack pane
      stackPane.getChildren().addAll(mapView, gridPane);
      StackPane.setAlignment(gridPane, Pos.TOP_LEFT);
      StackPane.setMargin(gridPane, new Insets(10, 0, 0, 10));
```